corehq/apps/accounting/forms.py

```diff
@@ -841,7 +841,7 @@ class ChangeSubscriptionForm(forms.Form):
     subscription_change_note = forms.CharField(
         label=gettext_lazy("Note"),
         required=True,
-        widget=forms.Textarea,
+        widget=forms.Textarea(attrs={"style": "resize:vertical"}),
```

Comment:
CSS classes are preferable to inline styles - could this and similar code be `attrs={'class': 'vertical-resize'}` instead?